pkg/handler/remote.go

```diff
@@ -61,10 +61,13 @@ func (h *remoteHandler) remoteAction(c *gin.Context, action string, rawQuery str
 	c.String(200, result)
 }
 
+// Select and return the datasource corresponding to the dsID or dsType parameter
 func (h *remoteHandler) getDatasourceWithParams(dsID string, dsType string) (model.Datasource, error) {
+	// Logic to return the first datasource if neither dsID nor dsType exists
 	if dsID == "" && dsType == "" {
 		dsID = "0"
```

Comment:
다시 보니 이렇게 dID="0"을 설정하는 것보다는 바로 리턴이 좋지 않을까요? 
```suggestion
		return h.datasourceStore.GetDatasourceByIndex(0)
```

코드 상단에서 어떤 값을 설정하고 그 설정한 값에 따라서 또 if 분기로 판단하는 것이 반복되다 보니까 조금 이해가 어려웠던 것 같네요.

Reply:
일단은 kubectl get pod에 아무 것도 안적으면 default namespace가 선택되는 것 같은 느낌으로 작성했는데, 없어도 될 것 같네요. 말씀하신대로 수정했습니다.

alert이나 dashboard에서 사용할 것 같은 datasource구분은 main/sub/all × lethe/prometheus/all 최대 이 정도 경우의 수가 있을 것으로 예상되네요.

일단 기존 버전의 alerting에서 사용가능하고 가능하고 있는 것은 main prometheus 밖에 없어요. 당장 필요한 것은 main lethe와 sub prometheus 정도네요. 필요에 따라 alert 모듈이나 dashboard에서 데이터소스 전체 목록을 조회하는 로직이 있으면 되니까 사실 getAll과 getByID만 있어도 될 것 같긴 하고요. 구체적인 건 그쪽을 개발하면서 정리하는 게 좋겠습니다.

Reply:
바꿔봤더니 테스트 코드가 실패하네요. 테스트 코드에는 dsid나 dstype을 기입하지 않았기 때문에 다 적어줘야 하는데... 원복하는 건 어떨까요? ㅎㅎㅎ

Reply:
일단은 테스트 코드에 datasource가 하나라서 suggestion 처럼
```
return h.datasourceStore.GetDatasourceByIndex(0)

으로 바꿔도 테스트 코드는 통과하네요. 일단 원복하셔도 좋고 todo나 issue로 기록해놓으면 좋을 것 같습니다.

Reply:
remote_test.go TestMedata, TestQuery, TestQueryRange 함수의 쿼리스트링에 `dsid=0` 추가해서 테스트 코드 통과시켰습니다... ㅎㅎ